.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
remove this before getting this merged to avoid production deployments caused by creation of pull request